src/components/ExpandedNode/ExpandedNode.tsx

```diff
@@ -312,6 +318,13 @@ function renderGuidance(
         }
         break;
       }
+      case 'CarePlan': {
+        const plan = documentation.resource as CarePlan;
+        const title = plan.title;
+        if (title) {
+          returnElements.push(<ExpandedNodeField key="Title" title="Title" description={title} />);
```

Comment:
should the title property actually be "Title" or should it be `title`?  Looks weird to me.

```suggestion
          returnElements.push(<ExpandedNodeField key="Title" title={title} description={title} />);
```

Reply:
Wouldn't that make the title and description the same thing?

Reply:
the title is more like the key, and the key is just the key.

Reply:
So looking at the `ExpandedNodeField` I see that title is really the field's title.  I'm going to assume that we'll never be pushing more than one `ExpandedNodeField` of this kind (i.e. with the same key).

https://github.com/mcode/pathways/blob/d4dd29451472d7bf840a2c3e1643fcfd30abbbea/src/components/ExpandedNode/ExpandedNode.tsx#L120-L127